.changeset/brown-islands-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops': patch
+---
+
+Fixed **AssignedToCurrentUsersTeams** & **CreatedByCurrentUsersTeams** filter in **AzurePullRequestsPage** component.
```

Comment:
You can just use backticks ` for this, looks a bit nicer: 

Fixed `AssignedToCurrentUsersTeams` & `CreatedByCurrentUsersTeams` filter in `AzurePullRequestsPage` component.

Reply:
Done 😄